Header header

string io_type

SeekCameraIOProperties io_properties

string chipid
string serial_number
string firmware_version

ThermographyWindow thermography_window

string color_palette
string agc_mode
string shutter_mode
string temperature_unit

float32 scene_emissivity
float32 thermography_offset